meta-mender-core/recipes-mender/mender-client/mender_4.x.inc

```diff
@@ -0,0 +1,5 @@
+
+# The docker module depends on bash, and of course on docker. However, docker is
+# a very large requirement, which we will not mandate. Bash however, we require,
+# because otherwise the Yocto QA checks will complain.
+RDEPENDS:mender-update:append = "${@bb.utils.contains('PACKAGECONFIG', 'modules', ' bash', '', d)}"
```

Comment:
Note to self,

On `kirkstone`, this needs to be added for 3.x recipes too.

Reply:
False alarm, 3.x recipes use `mender-client-go.inc` which was not modified (iow has this depenendy).